docs/openapi/paths/activity-reports/review.yaml

```diff
@@ -0,0 +1,40 @@
+put:
+  tags:
+    - activity-reports
+  summary: Review an activity report
+  description: >
+    An approving manager reviews an activity report to determine if it requires
+    any additional updates. If the report needs updates the manager sets the status to
+    'Needs Action', otherwise to 'Approved'
+  requestBody:
+    description: The status and any manager notes
+    required: true
+    content:
+      application/json:
+        schema:
+          type: object
+          properties:
+            status:
+              type: string
```

Comment:
Should we include an enum since we know there's a set of values that will be accepted?